Sources/VGSCollectSDK/UIElements/Text Field/VGSCardTextField.swift

```diff
@@ -12,12 +12,72 @@ import UIKit
 
 /// An object that displays an editable text area. Can be use instead of a `VGSTextField` when need to detect and show credit card brand images.
 public class VGSCardTextField: VGSTextField {
+    public enum SideCardIcon {
+        case left(size: CGSize)
```

Comment:
I should also add left/rigth cases without setting the size. So we can have 4 cases: [left, right, leftWithSize, rightWithSize]